ml_ops.sh

```diff
@@ -59,77 +59,47 @@ LDA_OUTPUT_DIR=${DSOURCE}/${FDATE}
 
 TOPIC_COUNT=20
 
+nodes=${NODES[0]}
```

Comment:
Didn't want to have the logic of creating result folders in ml_ops.sh so I moved the code inside the lda wrapper. It turned out I couldn't pass a list _space separated_ nodes because spark-submit/scala was taking each node as a new parameter for the .jar file.

I'm creating a new list nodes to pass node list _comma separated_. 
In my defense, I Didn't want to change duxbay.conf so I managed everything in ml_ops.sh.